go.mod

```diff
@@ -73,12 +74,12 @@ require (
 	google.golang.org/genproto/googleapis/rpc v0.0.0-20240123012728-ef4313101c80
 	google.golang.org/grpc v1.62.1
 	google.golang.org/protobuf v1.33.0
-	k8s.io/apimachinery v0.29.2
-	k8s.io/client-go v0.29.2
-	k8s.io/component-base v0.29.2
-	k8s.io/cri-api v0.30.0-alpha.3
+	k8s.io/apimachinery v0.29.3
+	k8s.io/client-go v0.29.3
+	k8s.io/component-base v0.29.3
+	k8s.io/cri-api v0.31.0-alpha.0
```

Comment:
(Probably we should rather use [v0.30.0-rc.0](https://github.com/kubernetes/cri-api/releases/tag/v0.30.0-rc.0))

Reply:
Manually modified the PR to use v0.30.0-rc.0